user/pages/03.commerce2/02.developer-guide/04.customers/02.addresses/03.address-entry/docs.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Address entry
+taxonomy:
+    category: docs
+---
+
+This documentation page describes how you can customize address fields as they appear on forms.
+
+* [Setting field overrides](#setting-field-overrides) describes how to control which address fields appear on forms and whether they are required.
+* [Altering address fields on forms](#altering-address-form-fields) describes how to change field labels and make other alterations.
+* [Setting initial values for address fields](#setting-initial-values-for-address-fields) describes how to set default values for address fields.
+
+### Setting field overrides
+For each address field (First name, middle name, last name, etc.), you can specify an *override* setting. By default, the data in the Commerce Guys *Addressing* library is used to determine how fields should be used for a specific country.
```

Comment:
It may help to refer to the individual fields with an address field as elements; the "field" in Drupal parlance really is the whole thing ... at least from a UI based configuration standpoint. (In code it would be a custom element type if I'm not mistaken.)

Reply:
I'm not sure if it should be elements or perhaps "properties", since a field "element" is a thing, too. This bothered me, too. I used "fields" because of the "Field overrides" UI. The column for the field elements/properties is labeled "Field". So I was trying to be consistent with that even though it's really the address as a whole that's the field. I'll ping Bojan about it.